package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "response",
-  "version": "0.18.1",
+  "version": "0.18.2",
```

Comment:
Back this out, release automation updates this.

Reply:
done!